packages/amplify-graphql-index-transformer/src/graphql-index-transformer.ts

```diff
@@ -112,12 +113,10 @@ function validate(config: IndexDirectiveConfiguration, ctx: TransformerContextPr
       }
 
       if (peerDirective.name.value === 'primaryKey') {
+        const hasSortFields = peerDirective.arguments!.some((arg: any) => arg.name.value === 'sortKeyFields');
```

Comment:
Does it accommodate if sortKeyFields explicitly set as empty `sortKeyFields: []`?

Reply:
Yes, it does.

Reply:
This schema failed to compile.
https://github.com/aws-amplify/amplify-cli/blob/abd8c67f7f4b76ece729f34447daf530179e1a1a/packages/amplify-util-mock/src/__e2e_v2__/model-relational-with-key-transformer.e2e.test.ts#L61-L66

Reply:
Thank you for your response, I was able to figure out what was going wrong. The tests should pass now.